### DISCUSSION TITLE
Where can I find documentation on searching/selecting/matching strings of text with "/" or "space+/" keybind?

### DISCUSSION THREAD

**Discussion by Kirirou** [helix-editor/helix#10768] [Q&A] [answered]
I'm new to this editor and was having trouble finding what kind of language or pattern system/syntax is used in the searching/matching/selecting function of the helix editor. 

I noticed that there are some special symbols can be used to match/select more effectively like "|" for OR and "$" and some characters like parenthesis have to be escaped with "\\" etc.

I tried searching the documentation on this, but couldn't easily find it.

Would really appreciate your help!

**Comment by kirawi** [comment, marked as answer]
It's called regex. You can see https://docs.rs/regex/latest/regex/#syntax for the engine we use. You can also view other regex resources (there are tons) since all regex implementations that I'm aware of have fairly similar syntax.

You can also check resources like https://regex101.com/ (which supports Rust regex as an option) to have a playground to learn in.
